abnf/odata-abnf-construction-rules.txt

```diff
@@ -220,7 +220,7 @@ functionImportCallNoParens     = entityFunctionImport
                                / primitiveFunctionImport
                                / primitiveColFunctionImport
 
-functionParameters = OPEN [ functionParameter *( COMMA functionParameter ) ] CLOSE
+functionParameters = OPEN [ BWS functionParameter *( BWS COMMA BWS functionParameter ) ] BWS CLOSE

```

Comment:
Functions like `concat` also allow BWS inside parentheses and around commas:
https://github.com/oasis-tcs/odata-abnf/blob/46b703893521001e631e015139901607a38e4e73/abnf/odata-abnf-construction-rules.txt#L629